firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -357,6 +357,13 @@ class BaseTestCase: XCTestCase {
             nrOfSwipes += 1
         }
     }
+
+    func panScreen() {
```

Comment:
I think this depends on the simulator used, let's add a comment with the one used

Reply:
updated